src/Shouldly.Tests/ShouldBeEquivalentTo/ObjectScenario.cs

```diff
@@ -81,6 +81,51 @@ but was
     but was
 ""Bob""
 
+Additional Info:
+    Some additional context");
+        }
+
+        [Fact]
+        public void ShouldFailWhenTitleDoesNotMatch()
```

Comment:
Could you name this `ShouldFailWhenFieldDoesNotMatch` to describe the feature being tested?